modules/react/tooltip/lib/OverflowTooltip.tsx

```diff
@@ -99,15 +99,17 @@ export const OverflowTooltip = ({
   children,
   ...elemProps
 }: OverflowTooltipProps) => {
-  const titleText = innerText(children);
-  const {targetProps, popperProps, tooltipProps} = useTooltip({type: 'label', titleText});
+  const [titleText, setTitleText] = React.useState('');
+  const {targetProps, popperProps, tooltipProps} = useTooltip({type: 'muted'});
```

Comment:
`muted` is the appropriate type in this case. This means a screen reader will not have any special affordances for overflow tooltips since overflowing is a visual concern and the calculated "name" of the element should already be known to assistive technology. Leaving as `label` in this case will actually remove the name from assistive technology because the `aria-label` will be calculated as `''` because `titleText` is an empty string until the user interacts with the element, either via `focus` or `mouseenter` events. `OverflowTooltip` is meant to be a visual aid when text is overflowed.